help_to_heat/frontdoor/consts.py

```diff
@@ -169,10 +169,12 @@
 duplicate_uprn_field = "uprn_is_duplicate"
 epc_found_field = "epc_found"
 lmk_field = "lmk"
+lmk_field_enter_manually = "enter-manually"
```

Comment:
Maybe not worth moving them now, but given they're only used within the class in question I don't think they need to be pulled out to this level (and having one global 'all constants' file is kind of messy)

Reply:
perhaps the constants file could be reviewed (the constants are a bit inconsistently spread between multiple files); could raise a backlog ticket to look into it

in this case since this is a form answer it will be passed through to the routing functions and so could feasibly be imported & reasoned with there, so I think this should be hoisted

Reply:
have raised as [PC-1379](https://beisdigital.atlassian.net/browse/PC-1379)